### DISCUSSION TITLE
Create install script with openmptcprouter

### DISCUSSION THREAD

**Discussion by onlineapps-cloud** [tteck/Proxmox#2145] [Ideas]
hi, first i want to say thanks for your work, very useful scripts to deploy useful tools.
It's possible to create VM with this system like Openwrt? https://www.openmptcprouter.com/
On download page at the end, persist different x86-64 (64-bits) OS types, thanks, best whishes.

**Comment by tteck** [comment]
In their [documentation](<https://github.com/Ysurac/openmptcprouter/wiki/Router-install#for-proxmox>)
> For Proxmox
You can use this tutorial, by replacing OpenWrt image by OpenMPTCProuter image: https://www.jwtechtips.top/how-to-install-openwrt-in-proxmox/

**Comment by onlineapps-cloud** [comment]
i already install openmptcprouter on proxmox, but i this image don't have DHCP enabled to get ip for my local network, as your version of openwrt, and i don't know what i need to do, to enable DHCP, i don't familiarized with openwrt, my local network cidr is `192.168.1.0/24 on vmbr0` and `10.10.161./24 on vmbr2`
![image](https://github.com/tteck/Proxmox/assets/69077240/3241dc2f-0de0-44ae-acf2-6402a3140f42)
![image](https://github.com/tteck/Proxmox/assets/69077240/d4d6001d-3686-4e37-8c48-89306e6b4e6d)

**Comment by onlineapps-cloud** [comment]
i will try this:
```
uci set network.lan.ipaddr='x.x.x.x'
uci commit network
/etc/init.d/network restart
```
UPD: this worked, i able to login in system.
